std/array.d

```diff
@@ -2517,8 +2517,15 @@ Implements an output range that appends data to an array. This is
 recommended over $(D a ~= data) when appending many elements because it is more
 efficient.
  */
-struct Appender(A)
-if (isDynamicArray!A)
+alias Appender(A) = Buffer!(A, false);
```

Comment:
Either that, or an enum (or std.typecons.Flag) should be defined that can be optionally passed to the constructor.